### ISSUE TITLE
Document search by_field all fields *

### ISSUE THREAD

**Issue by CrashLaker** [typesense/typesense#270]
Hi all,

Is it possible to `query_by` all fields?

something like

```python
search_parameters = {
  'q'         : 'stark',
  'query_by'  : '.*'
}

client.collections['companies'].documents.search(search_parameters)
```

or would i need to `retrieve` the collection fields' then place it in `query_by`?

https://typesense.org/docs/0.11.2/api/documents.html#search

Regards,
Carlos.

**Comment by kishorenc** [comment]
No, that's not possible. You will have to explicitly specify the list of fields in a comma separated string. E.g. `query_by=name,description,label`.

**Comment by tao** [comment]
If this was supported it would make using the javascript client a lot easier to use (in some cases) if you're working with multiple collections.

Instead of the website doing a query to fetch the field names... before you can search typesense... we could just perform a search with a wildcard and the typesense server could do an internal query to find the field names if they are missing.

**Comment by tharropoulos** [comment]
This has been fixed since https://github.com/typesense/typesense/pull/852 (v0.25)
CC: @jasonbosco @kishorenc
